stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivity.kt

```diff
@@ -164,6 +163,7 @@ class PaymentMethodsActivity : AppCompatActivity() {
             // Refresh the list of Payment Methods with the new reusable Payment Method.
             fetchCustomerPaymentMethods()
             viewModel.onPaymentMethodAdded(paymentMethod)
+            adapter.selectedPaymentMethodId = viewModel.selectedPaymentMethodId
```

Comment:
On second thought, it's more correct for this line to go before `fetchCustomerPaymentMethods()`, because that is what causes the RecyclerView to refresh its contents.

Reply:
👍